sphinxdocs/src/sphinx_bzl/bzl.py

```diff
@@ -498,7 +498,40 @@ def run(self) -> list[docutils_nodes.Node]:
         self.env.ref_context["bzl:file"] = file_label
         self.env.ref_context["bzl:object_id_stack"] = []
         self.env.ref_context["bzl:doc_id_stack"] = []
-        return []
+
+        _, _, basename = file_label.partition(":")
+        index_description = f"File {label}"
+        absolute_label = repo + label
+        self.env.get_domain('bzl').add_object(
+                _ObjectEntry(
```

Comment:
Style: 8 spaces?

Reply:
Doh. Fixed; ran the pre-commit formatters